Examples/Java/Sources/Board.java

```diff
@@ -375,6 +377,21 @@ public Builder setUrl(@Nullable String value) {
 
         @NonNull
         public Board build() {
+            List<String> missingFields = new ArrayList<>();
```

Comment:
Minor: Do we need to use `List` here at all? We could just use `ArrayList<String> missingFields = new ArrayList<>()` because this is locally-scoped.

Reply:
Done.